researchstack-mpower/src/main/java/org/sagebionetworks/research/mpower/researchstack/framework/MpTaskHelper.java

```diff
@@ -109,9 +113,6 @@ protected Task createSmartSurveyTask(Context context, @Nullable TaskModel taskMo
     @Override
     protected void addFiles(Archive.Builder archiveBuilder, List<Result> flattenedResultList, String taskResultId) {
 
-        // We also upload each individual json answer result to support AppCore surveys
-        super.addFiles(archiveBuilder, flattenedResultList, taskResultId);
```

Comment:
This was a bug where results were being added twice, as this is also called again in the function below.